src/main/resources/messages/messages.properties

```diff
@@ -107,18 +107,24 @@ error.translation_not_exists='%s' translation file doesn't exist in the specifie
 error.upload_translation_to_storage=Failed to upload the translation file to the storage. Please contact our support team for help
 error.upload_translation=Failed to upload the translation file. Please contact our support team for help
 error.configuration_file_is_invalid=Configuration file is invalid. Check the following parameters in your configuration file:
+error.params_are_invalid=Params are invalid. Check the following parameters:
 error.empty_properties_file=Configuration file is empty. Fill it out accordingly to the following requirements: \
 https://support.crowdin.com/configuration-file/#configuration-file-structure
 error.configuration_file_not_exist=Configuration file doesn't exist. Run the 'crowdin init' to generate configuration skeleton
+error.identity_file_not_exist=Specified identity file doesn't exist
 error.reading_configuration_file=Failed to read the configuration file. Try to run the application with admin permission
 error.clearing_temp=Failed to clear temporary files
 error.while_checking_base_path=Failed to check base path. Try to run the application with admin permission
 
+error.response.organization_not_found=Organization not found
+error.response.project_not_found=Project with id '%s' doesn't exist
+error.response.unauthorized=Please make sure you entered valid Personal Access Token
```

Comment:
Failed to authorize in Crowdin. Make sure you entered valid Personal Access Token or try to generate a new one